sprint8/encyclopedia1.js

```diff
@@ -0,0 +1,20 @@
+/*
+  Local Storage
+  Session Storage
+
+  Jaka jest różnica pomiędzy local storage a session storage?
+*/
+
+/*
+  Local Storage (pamięć lokalna) to rodzaj pamięci internetowej, który umozliwia stronom  i aplikacjom przechowywanie danych. Oznacza to, ze przechowuje obiekt danych bez datych wazności. Dane nie będą usuwane po zamknięciu przeglądarki i będzie dostępna następnego dnia, tygodnia lub roku.
```

Comment:
Nie tłumaczmy po polsku :) zostańmy przy angielskiej nazwie `localStorage`
` ... ze przechowuje obiekt danych bez datych wazności` - nie wiem co to znaczy :(
`Dane nie będą usuwane po zamknięciu przeglądarki i będzie dostępna następnego dnia, tygodnia lub roku.` - trochę nie po polsku, `Dane nie będą ... i bęDĄ DOSTĘPNE` - tak powinno być